express/views/partials/head.ejs

```diff
@@ -1,6 +1,7 @@
 <meta charset="utf-8" />
 <meta http-equiv="X-UA-Compatible" content="IE=edge" />
 <meta name="viewport" content="width=device-width, initial-scale=1" />
+<meta http-equiv="Cache-Control" content="no-sleep">
```

Comment:
What's `no-sleep`? I don't see that option documented anywhere here:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
I just added it to get `cache-controls-missing` firing as a POC.  I plan on following up with formal tickets to hit some of our missing rules

Reply:
Gotcha, so an intentionally invalid value.